### DISCUSSION TITLE
BFV Bootstrapping

### DISCUSSION THREAD

**Discussion by czh-rot** [tuneinsight/lattigo#429] [Q&A]
I was doing some cryptographic operations using BFV, however, I found that for N=2^15, I could only do 17 multiplications. How should I implement bootstrapping? It looks like this function isn't implemented in the repository.

**Comment by Pro7ech** [comment]
Dear @czh-rot ,

Thanks for your question. We actively push forward new functionalities and features on Lattigo prioritized according to use cases and needs from its users. Please, send us an email to [lattigo@tuneinsight.com](mailto:lattigo@tuneinsight.com) with more context information about the use case for <functionality> in your setting, and whether it is intended for academic or industrial use. This information can help us in giving you a more informed response about your request, and also to shape the future development of Lattigo accounting to the requirements of the community.

Best,
Jean-Philippe
